dgraph/cmd/alpha/run.go

```diff
@@ -208,6 +208,10 @@ they form a Raft group and provide synchronous replication.
 				"worker in a failed state. Use -1 to retry infinitely.").
 		Flag("txn-abort-after", "Abort any pending transactions older than this duration."+
 			" The liveness of a transaction is determined by its last mutation.").
+		Flag("shared-instance", "When set to true, it disables ACLs for non-galaxy users. "+
+			"It expects the access JWT to be contructed outside dgraph for those users as even "+
```

Comment:
constructed*
We need to improve the language. I am not sure what this line means.
`t expects the access JWT to be contructed outside dgraph for those users as even login is denied to them`

Reply:
This is still not addressed yet

Reply:
I have changed the language (added another line just after. Let me know if you have a suggestion?

Reply:
I think the purpose of this flag is not so that the customer use external JWT libraries, "This can be done via online tools or jwt libraries". I am not sure how this helps.